sdk/src/main/java/com/mapbox/maps/viewannotation/ViewAnnotation.kt

```diff
@@ -16,6 +16,10 @@ internal data class ViewAnnotation(
    * if user did not specify [ViewAnnotationOptions.visible]  explicitly  during add / update operation
    */
   var handleVisibilityAutomatically: Boolean,
+  /**
+   * Cached value if view was visible before OnGlobalLayoutListener was triggered.
+   */
+  var wasVisible: Boolean,
```

Comment:
what is the default value?
can user change this variable?

Reply:
this class is internal, we control it ourselves
see no sense in setting default value